exercises/ex_atm.py

```diff
@@ -0,0 +1,137 @@
+# ATM Exercise
+
+import sys
+
+FILE_PATH = 1
+PASSWORD_INDEX = 0
+BALANCE_INDEX = 1
+
+
+def read_file_move_to_dict(file_path):
+    """Read file and move atm information to dictionary.
+
+    Args:
+        file_path (string): path to atm information file.
+
+    Return:
+        A dictionary, the key is each users id.
+        the value is a list of user's password and balance.
+    """
+    try:
+        atm_options_dict = {}
+        with open(file_path, 'r') as file_pointer:
+            for line in file_pointer:
+                line_list = line.rstrip().split(',')
+                atm_options_dict.update({line_list[0]: [line_list[1],
+                                                        float(line_list[2])]})
+        return atm_options_dict
+
+    except FileNotFoundError:
+        print("The path to the file you have entered was incorrect.")
+
+    except Exception as e:
+        print(e)
+
+
+def check_input_execute(atm_dict, user_input, user_id):
+    """Check user option input, and act accordingly, if input is correct.
+
+    Args:
+        atm_dict (dictionary): the dictionary, containing all the atm
+                               information.
+        user_input (string): the option choice of the user.
+        user_id (string): the id of the user, to find him in the dictionary.
+
+    Return:
+        The new changed dictionary of atm information.
+    """
+    try:
+        # ['check', 'withdrawal', 'deposit', 'change']:
+        if user_input == "check":
+            print(f"You'r balance: {atm_dict[user_id][BALANCE_INDEX]}")
+
+        elif user_input == "withdrawal":
+            withdrawal = input("Enter how much you want to withdrawal -> ")
+            if float(withdrawal) > 0:
```

Comment:
If a user entered a negative amount, nothing will happen and the user will not even know it. output something to the user telling him his input wasn't correct.